File: CSharp.Functions/Common/Entities/FirmDetails/Locum/LocumSoleTraderDetails.cs
namespace Common.Entities
{
    public class LocumSoleTraderDetails
    {
        public string? ForeName { get; set; }
        public string? SurName { get; set; }
        public string? FirmReferenceNumber { get; set; }
        public string? TradingAddress { get; set; }
        public bool IsTradingAddressChanged { get; set; }
        public string? EmailAddress { get; set; }
        public ContactNumber? ContactNumber { get; set; }
        public string? Website { get; set; }
    }
}

File: vue.jsonforms/src/entities/firm-details/LocumSoleTraderDetails.ts
import { ContactNumber } from "@/entities/ContactNumber";

export class LocumSoleTraderDetails {
  public foreName?: string;
  public surName?: string;
  public firmReferenceNumber?: string;
  public tradingAddress?: string;
  public isTradingAddressChanged: boolean = false;
  public emailAddress?: string;
  public contactNumber?: ContactNumber;
  public website?: string;
}